src/crypto/BUILD.gn

```diff
@@ -38,12 +38,24 @@ assert(
         chip_crypto == "platform",
     "Please select a valid crypto implementation: mbedtls, psa, openssl, boringssl, platform")
 
+if (chip_crypto_keystore == "") {
+  if (chip_crypto == "psa") {
+    chip_crypto_keystore = "psa"
+  } else {
+    chip_crypto_keystore = "raw"
+  }
```

Comment:
Add "provided_by_app" which includes zero new files and disables the unit tests for the keystore.